lib/commands/nw-test/index.js

```diff
@@ -34,6 +36,37 @@ module.exports = {
     this.quickTemp.remove(this, '-testsDistNW');
   },
 
+  TestServerBuilder: Builder.extend({
```

Comment:
yeah - I think I was just trying to avoid mucking with the application's build process all together. (this will impact `ember build`) I thought keeping the behavior just in `ember nw:test` would have been a good idea (for example changing `tests/index.html` file contents) but then again this was just a guess.

We can do it with the ember-cli add-on hooks. However a slight functional problem that I can't seem to figure out is how to hide or show nwjs. Right now the `tests/package.json` in the blue prints has a `window.show` property in its `package.json` file set to false (for a CI mode experience).

If i use `postprocessTree` the `package.json` file doesn't get copied before the the promise from `Builder.build()` (inside [nw-test index.js](lib/commands/nw-test/index.js) ) is resolved in the and so trying to change this flag (by reaching in the file and changing it via code) doesn't work. The `postprocessTree` hook also doesn't tell us what the command line that invoked the code so I don't think using the broccoli plugin will work either. So there's no way to discern from development and ci test experiences.

I was thinking about changing the source package.json... but I think that would cause a pending change on the file under source control. Open to ideas here...